content/blog/integrating-devops-and-security-for-scalable-platform-engineering/index.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Integrating DevOps and Security for Platform Engineering"
```

Comment:
removed "Scalable" to keep within 51 to 60 characters per [guidelines](https://docs.google.com/presentation/d/1OvdLR_25BXrAvozYt5XzMNGBVuy9a_NL4gsnEz1_uZA/edit#slide=id.g3164f0b88d5_0_35).